### ISSUE TITLE
Removal of Parabox Duplication

### ISSUE THREAD

**Issue by Tomstah** [Darkosto/SkyFactory4Issues#542]
## Context
For those that arn't already aware, the parabox upon collapsing a world _does not_ affect PlayerData.  This essentially means you can edit your inventory before collapsing a parabox, and your inventory will be the same afterwards.  Not only is this ["Intended Feature"](https://github.com/Darkosto/SkyFactory4Issues/issues/252) left unsaid, possibly causing item loss (I lost several items, including the original dragon egg due to the fact that I had **_assumed_** my inventory would be reverted).  This creates the opportunity to duplicate materials.  Not only does this allow players to essentially duplicate entire systems, harder to craft objects, etc, skipping automation, progression, and more, but for those that own servers this allows abuse of the system.

## TL;DR
The parabox needs to be changed so that it **_also_** reverts PlayerData upon collapsing (Preventing unnescessary item loss and duplication).  At the very least this needs to be an optional feature.

**Comment by WhyNotCthulhu** [comment]
This sounds more like something that should be on the mod GitHub rather than the modpack GitHub.

**Comment by justinrusso** [comment]
> I lost several items, including the original dragon egg due to the fact that I had assumed my inventory would be reverted
Not really sure what you mean by this. If you put it in the world or elsewhere, itd be reverted, so you would have lost it then. The only place you wouldnt have would be in your inventory.

In either case, this is a design decision and wont be changing. If you find this to be an issue on your server (even though thats literally one of the known intentions of this mod in the pack) then dont use it.
